File: java/secao8/Exercicio04.java
package br.com.geekuniversity.secao8;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		// variáveis
		int [] vetor = new int [20];
		int soma=0;
		Scanner teclado = new Scanner(System.in);
				
		//Entradas 
		for (int i=0; i<20; i++) {
		System.out.printf("Informe o vetor %d/20: \n", (i+1) );	
		vetor[i] = teclado.nextInt();
		
		//processamento
		soma = soma + vetor[i];
		}
		//saída 
		System.out.printf("A soma é %d", soma);
		teclado.close();
	}

}

File: python/python_secao8/Exercicio04.py
#variáveis
vetor = []
soma = 0

#entradas
for i in range(0, 20):
    num = int (input("Informe um número:{0}/20".format(i+1)))
    vetor.append(num)
    
#processamento
    soma = soma + num
#saída
print("{0}".format(soma))

